src/ssl.c

```diff
@@ -11057,7 +11057,24 @@ WOLFSSL_SESSION* wolfSSL_get_session(WOLFSSL* ssl)
     return NULL;
 }
 
-
+/*
+ * Sets the session object to use when establishing a TLS/SSL session using
+ * the ssl object. Therefore, this function must be called before
+ * wolfSSL_connect. The session object to use can be obtained in a previous
+ * TLS/SSL connection using wolfSSL_get_session.
+ *
+ * This function rejects the session if it has been expired when this function
+ * is called. Note That this expiration check is wolfSSL specific and it brigns
```

Comment:
Spelling - "brigns" should be "brings", but let's re-word that phrase to say "Note that this expiration check is wolfSSL specific and differs from OpenSSL return code behavior".

Reply:
Replaced the statements.